rdr_service/resource/constants.py

```diff
@@ -28,6 +28,8 @@ class SchemaID(IntEnum):
     participant_modules = 2070
     participant_address = 2080
     patient_statuses = 2090
+    ehr_recept = 2100
+    pdr_participant = 2010
```

Comment:
Same value as `participant_biobank_orders` on line 22?

Reply:
Good catch, I'll fix that.